ftw_tools/inference/model_registry.py

```diff
@@ -18,7 +18,7 @@ class ModelSpec(BaseModel):
 
     url: str
     description: str = Field(min_length=1)
-    license: Literal["CC BY 4.0", "proprietary", "AGPL-3"]
+    license: Literal["CC BY 4.0", "proprietary", "AGPL-3", "Mixed Open Licenses"]
```

Comment:
```suggestion
    license: Literal["CC BY 4.0", "AGPL-3", "Mixed Open Licenses"]
```

There's no use of "proprietary" left, right?